spec/support/transactions/operation.rb

```diff
@@ -114,12 +114,12 @@ def execute(collection, session0, session1)
         {
           'errorCodeName' => err_doc['codeName'] || err_doc['writeConcernError']['codeName'],
           'errorContains' => e.message,
-          'errorLabels' => (e.instance_variable_get(:@labels) || []) + (err_doc['errorLabels'] || [])
+          'errorLabels' => e.instance_variable_get(:@labels)
         }
       rescue Mongo::Error => e
         {
           'errorContains' => e.message,
-          'errorLabels' => e.instance_variable_get(:@labels) || []
+          'errorLabels' => e.instance_variable_get(:@labels)
```

Comment:
I think Mongo::Error should have a `labels` method added following this part of https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#error-reporting-changes:

> Drivers MAY expose the list of all error labels for an exception object.

Reply:
Will do